src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -151,88 +150,79 @@ def generate_ssns(
     :param size: The number of itins to generate
     :param additional_key: Additional key to be used by randomness
     :param randomness: RandomnessStream stream to use
-    :param allow_duplicates: True if duplicates are allowed, False (default) otherwise
-
     """
+    ssn = pd.DataFrame(index=pd.Index(range(size)))
 
-    def generate_ssn(
```

Comment:
Since we no longer need to check for duplicates, this internal function that was previously being called in a while loop (while there were duplicates) is no longer necessary - all of this change is just removing the function definition and then de-indenting (outdenting?)